### ISSUE TITLE
Distorted audio in stream or recording if OBS is started before the game I record.

### ISSUE THREAD

**Issue by PaddyMac** [obsproject/obs-studio#10916]
### Operating System Info

Other

### Other OS

Gentoo Linux

### OBS Studio Version

30.2.0-beta4

### OBS Studio Log URL

https://obsproject.com/logs/HEtkg79ONiTYBa91

### Expected Behavior

I expect audio to be streamed or recorded normally (i.e. without distortion) whether OBS is started before or after a game I record.

### Current Behavior

OBS seems to work fine if I start a game first then start OBS and begin streaming or recording. But if I start OBS first and then start a game and begin streaming or recording, the audio is severely distorted.

### Steps to Reproduce

1. Start OBS
2. Start a game
3. Stream or record
...

### Anything else we should know?

I'm currently using 30.2.0-beta4, but this has been an issue for me at least since the 30.1.0 release (though I don't recall exactly when it started).

**Comment by kkartaltepe** [comment]
I cannot reproduce this with the steps given, you are also using third party plugins for audio capture it seems. If you can remove your third party plugins and reproduce this we can consider reopening.

You should probably ensure that your audio is not being reconfigured while obs is open which we do not support and could lead to issues. Pipewire has been seen to do this during start of games for reasons im sure only they could answer.

**Comment by PaddyMac** [comment]
The closest I had to any kind of audio plugin was something to produce subtitles on the fly from my voice, but the issue started before I installed that. Nonetheless, I removed all three plugins I had installed and still have the error.

Here is a log produced while the error occurred: https://obsproject.com/logs/UUJCzpa63bKyV2IV
Here is a log file produced only moments later without the error: https://obsproject.com/logs/JOqEsjgO1eH4Gdgw

I should also add that audio from my mic seems unaffected. It's only the game audio.
